dbt/adapters/athena/impl.py

```diff
@@ -100,6 +100,7 @@ class AthenaConfig(AdapterConfig):
         partitions_limit: Maximum numbers of partitions when batching.
         force_batch: Skip creating the table as ctas and run the operation directly in batch insert mode.
         unique_tmp_table_suffix: Enforce the use of a unique id as tmp table suffix instead of __dbt_tmp.
+        tmp_schema: Define in which schema to create temporary tables used in incremental runs.
```

Comment:
If the tmp schema must be created by the user please specify that in the readme. Otherwise consider to can the create schema macro to create the schema if does not existm